### ISSUE TITLE
CLI throws unhandled error when I'm trying to generate an angular application.

### ISSUE THREAD

**Issue by eziotobioli** [single-spa/create-single-spa#169]
**Describe the bug or question**
I haven't been able to create an application through the cli due to an unhandled error, the app is generated correctly, the packages are installed but when the process is finishing throws this un handled error.

**To Reproduce**
For bugs, create a small git repo or codepen that reproduces the problem, and provide steps on how to see the issue. This is also helpful for questions.
Will noy attach codepen due to the error being on the generation of the app.

**Expected behavior**
Generate an angular app configured for single-spa with the cli.

**Screenshots and/or console output**

```
events.js:292
      throw er; // Unhandled 'error' event
      ^

TypeError: Cannot destructure property 'dependencies' of '(intermediate value)' as it is undefined.
    at SingleSpaAngularGenerator.validateSingleSpaDependency (C:\Users\<user>\node_modules\generator-single-spa\src\angular\generator-single-spa-angular.js:97:13)
Emitted 'error' event on SingleSpaAngularGenerator instance at:
    at Immediate.<anonymous> (C:\Users\<user>\node_modules\yeoman-generator\lib\index.js:1007:20)
    at processImmediate (internal/timers.js:456:21)
```

**Additional context**

the versions being used are these:

![imagen](https://user-images.githubusercontent.com/26915984/90831457-bd69d400-e311-11ea-9c97-57fd5a4eb323.png)

**Comment by joeldenning** [comment]
Hi @eziotobioli, thanks for reporting this. I have transferred this to the create-single-spa repo. Here is the line of code throwing the error:

https://github.com/single-spa/create-single-spa/blob/f47835287e2dbef823724dd77d78d61341fa9886/packages/generator-single-spa/src/angular/generator-single-spa-angular.js#L97

One thing to note is that single-spa-angular does not yet fully support Angular 10. There are several issues in the single-spa-angular repo that discuss it. If you try with Angular 9, I think that this may work.

**Comment by eziotobioli** [comment]
Hello Joel, thanks for your quick response, I wrote the wrong versions.

This is happening with andular 9.1.12.

![imagen](https://user-images.githubusercontent.com/26915984/90831284-6532d200-e311-11ea-825d-617e9ca898e6.png)

I've tried installing and uninstalling multiples times the CLI, and still not able to create applications.

Sorry for the misleading issue, hopefully now I get the versions right.

**Comment by filoxo** [comment]
Can you provide any additional steps on how to reproduce this issue? I tested creating an Angular app with the latest version of create-single-spa using those exact versions of node and \@angular/cli on both Mac and Windows, and they succeed without issue.

**Comment by eziotobioli** [comment]
Sorry for the late response, what else can i provide you to help? the steps i do are basically install globally with npm the CLI and proceed to create an app without any arguments selecting angular app through the wizard, then select application/parcel and last npm as package manager.

Thanks beforehand for your help.

**Comment by filoxo** [comment]
Right, so I was able to successfully generate Angular applications using the same versions of node, angular-cli, and the latest version of create-single-spa (since you haven't provided which version your machine has) on both Windows and Mac platforms. Unfortunately without being able to reproduce I can only guess as to what causes the error you're experiencing. The error is basically saying that its trying to read the dependencies from the package.json that should've been generated by the angular-cli but it doesn't exist. My only theory is that the angular-cli is failing silently but that's just a guess. Here are a bunch of long-shot questions:

- What version of create-single-spa is installed on your machine? `npm ls -g create-single-spa` should give you this
- Do you have a proxy from which you are getting packages (eg. artifactory or similar)? This might be something in an `.npmrc` file on your machine somewhere
- Which version of npm are you using? `npm -v` should give you this
- How are did you install node? 
- Do you use nvm-windows or similar to manage multiple versions of node?
- Do other Windows users at your company/org experience this same issue?
- What directory are you running `create-single-spa` from? A theory is that that directory that doesn't allow you to write to it.

**Comment by deggesim** [comment]
Same exact issue here.

![angular_cli](https://user-images.githubusercontent.com/9460787/92608239-64a8ae00-f2b5-11ea-921c-1b9e27e8301a.jpg)

I answer to your questions @filoxo 
create-single-spa 1.12.4
No proxy
npm version 6.14.6
node installed by Nodesouce binary distributions https://github.com/nodesource/distributions

I found that the issue is when you don't provide the directory for the project.

**Comment by filoxo** [comment]
Thanks for the additional info @deggesim. I'll take a look in the next day or two.

**Comment by filoxo** [comment]
This fix was released in [create-single-spa@1.12.6](https://github.com/single-spa/create-single-spa/releases/tag/v1.12.6).
